MidasCivil_Adapter/Convert/ToMidasCivil/Properties/FromMetaData.cs

```diff
@@ -0,0 +1,92 @@
+        {
+            StringBuilder midasMetaDataSB = new StringBuilder();
+            List<Review> reviews = midasMetaData.Reviews;
+            midasMetaDataSB.AppendLine("*PROJINFO");
+            midasMetaDataSB.AppendLine($"PROJECT={midasMetaData.ProjectNumber}");
+            midasMetaDataSB.AppendLine($";DESIGNSTAGE={midasMetaData.DesignStage}");
+            midasMetaDataSB.AppendLine($";PROJECTLEAD={midasMetaData.ProjectLead}");
+            midasMetaDataSB.AppendLine($"REVISION={midasMetaData.Revision}");
+            midasMetaDataSB.AppendLine($"USER={midasMetaData.Author}");
+            midasMetaDataSB.AppendLine($"EMAIL={midasMetaData.Email}");
+            midasMetaDataSB.AppendLine($"ADDRESS={midasMetaData.Location.Replace(System.Environment.NewLine, ", ")}");
+            midasMetaDataSB.AppendLine($"CLIENT={midasMetaData.Client}");
+            midasMetaDataSB.AppendLine($"TITLE={midasMetaData.ProjectName}");
+            midasMetaDataSB.AppendLine($"ENGINEER={midasMetaData.Author}");
+            midasMetaDataSB.AppendLine($"EDATE={midasMetaData.CreationDate.ToString("yyyy-MM-dd")}");
+            midasMetaDataSB.AppendLine($";DESCRIPTION={midasMetaData.Description}");
+            midasMetaDataSB.AppendLine($";DISCIPLINE={midasMetaData.Discipline}");
+
+            if(reviews.Count > 4)
+            {
+                reviews.RemoveRange(1, reviews.Count - 2);
+                Engine.Reflection.Compute.RecordWarning("The number of reviews exceeds the limit of MidasCivil. Only the first, last and second to last reviews will be recorded.");
+            }
+            if(reviews.Count >= 1)
+            {
+                midasMetaDataSB.AppendLine($"CHECK1={reviews[0].Reviewer}");
+                midasMetaDataSB.AppendLine($"CDATE1={reviews[0].ReviewDate.ToString("yyyy-MM-dd")}");
+            }
+            if (reviews.Count >= 2)
+            {
+                midasMetaDataSB.AppendLine($"CHECK2={reviews[1].Reviewer}");
+                midasMetaDataSB.AppendLine($"CDATE2={reviews[1].ReviewDate.ToString("yyyy-MM-dd")}");
+                Engine.Reflection.Compute.RecordWarning("Only comments from the last review will be pushed to MidasCivil.");
+            }
+            if (reviews.Count >= 3)
+            {
+                midasMetaDataSB.AppendLine($"CHECK3={reviews[2].Reviewer}");
+                midasMetaDataSB.AppendLine($"CDATE3={reviews[2].ReviewDate.ToString("yyyy-MM-dd")}");
+            }
+            if(reviews[reviews.Count - 1].Approved)
+            {
+                midasMetaDataSB.AppendLine($"APPROVE={reviews[reviews.Count - 1].Reviewer}");
+                midasMetaDataSB.AppendLine($"ADATE={reviews[reviews.Count - 1].ReviewDate.ToString("yyyy-MM-dd")}");
+            }
+
+            foreach(string comment in reviews[reviews.Count - 1].Comments)
+            {
+                midasMetaDataSB.AppendLine($"COMMENT={comment}");
```

Comment:
I wonder if some of the other information can be kept in the comments, such as the discpline, description etc.

Also, there should be a check to make sure it doesn't go over the character limit, and raise a warning if it does.